docs/02.API-REFERENCE.md

```diff
@@ -9171,6 +9171,52 @@ jerry_set_realm (jerry_value_t realm_value);
 
 - [jerry_create_realm](#jerry_create_realm)
 
+## jerry_realm_get_this
+
+**Summary**
+
+Gets the 'this' binding of a realm. The 'this' binding is always an object.
+By default the 'this' binding is the same as the realm object and can be
+changed by [jerry_realm_set_this](#jerry_realm_set_this).
+
+*Notes*:
+- This feature depends on build option (`JERRY_BUILTIN_REALMS`) and can be checked
+  in runtime with the `JERRY_FEATURE_REALM` feature enum value,
+  see: [jerry_is_feature_enabled](#jerry_is_feature_enabled).
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_realm_get_this (jerry_value_t realm_value)
+```
+- `realm_value` - realm value
+- return
+  - exception - if any error is happened
```

Comment:
If I understand correctly only TypeError can be returned if the input value is not a correct realm object. This can be described here.